src/main/java/WayofTime/bloodmagic/command/sub/SubCommandNetwork.java

```diff
@@ -2,182 +2,215 @@
-                    network.setCurrentEssence(maxOrb);
-                    CommandBloodMagic.displaySuccessString(sender, "commands.bloodmagic.network.cap.success", player.getDisplayName().getFormattedText());
-                }
-            }
-        },;
+        @Override
+        public void execute(MinecraftServer server, ICommandSender sender, String[] args) throws CommandException {
+            this.player = args.length < 1 ? getCommandSenderAsPlayer(sender) : getPlayer(server, sender, args[0]);
+            sender.sendMessage(new TextComponentString((player != sender ? player.getDisplayName().getFormattedText() + " " : "" + new TextComponentTranslation("tooltip.bloodmagic.sigil.divination.currentEssence", network.getCurrentEssence()).getFormattedText())));
+        }
+    }
 
-        public String help;
+    class Cap extends NetworkCommand {
 
-        ValidCommands(String help) {
-            this.help = help;
+        @Override
+        public String getName() {
+            return "cap";
         }
 
-        public abstract void run(EntityPlayer player, ICommandSender sender, boolean displayHelp, String... args);
+        @Override
+        public void execute(MinecraftServer server, ICommandSender sender, String[] args) throws CommandException {
+            super.execute(server, sender, args);
+            network.setCurrentEssence(NetworkHelper.getMaximumForTier(network.getOrbTier()));
+            sender.sendMessage(new TextComponentTranslation("commands.bloodmagic.network.cap.success", player.getDisplayName().getFormattedText()));
+        }
+    }
+
+    class Fill extends NetworkCommand {
+
+        @Override
+        public Integer getInfo() {
+            return Integer.MAX_VALUE;
+        }
+
+        @Override
+        public String getName() {
+            return "fill";
+        }
+
+        @Override
+        public void execute(MinecraftServer server, ICommandSender sender, String[] args) throws CommandException {
+            super.execute(server, sender, args);
+            network.setCurrentEssence(Integer.MAX_VALUE);
+            sender.sendMessage(new TextComponentTranslation("commands.bloodmagic.network.fill.success", player.getDisplayName().getFormattedText()));
+        }
+    }
+
+    class Tickets extends NetworkCommand {
+
+        @Override
+        public String getName() {
+            return "tickethistory";
+        }
+
+        @Override
+        public void execute(MinecraftServer server, ICommandSender sender, String[] args) throws CommandException {
+            super.execute(server, sender, args);
+            List<SoulTicket> tickethistory = network.getTicketHistory();
+            if (tickethistory.isEmpty())
+
+                for (SoulTicket i : network.getTicketHistory())
+                    sender.sendMessage(i.getDescription());
+            sender.sendMessage(new TextComponentTranslation("commands.bloodmagic.network.ticket.success", player.getDisplayName().getFormattedText()));
```

Comment:
Let the component handle the player name. Just provide `player.getDisplayName()`.